packages/patternfly-4/react-tokens/src/generateTokens.js

```diff
@@ -37,11 +37,17 @@ cssFiles.forEach(filePath => {
         });
         // Avoid stringifying numeric chart values
         const chartNum = decl.property.startsWith('--pf-chart-') && !isNaN(populatedValue);
+        const calculatedValue = chartNum ? Number(populatedValue).valueOf() : populatedValue;
         tokens[key] = {
+          ...tokens[key],
           name: property,
-          value: chartNum ? Number(populatedValue).valueOf() : populatedValue,
+          value: calculatedValue,
           var: `var(${property})`
         };
+        const isGlobal = populatedValue.indexOf('--pf-global') > -1;
```

Comment:
Nit: You can use `.includes()`  and spread into `tokens[key] = ` to reduce these 4 lines down to 1. Like
```js
tokens[key ] = {
  ...(populatedValue.includes('--pf-global') && { global: populatedValue })
}
```